providers/base/units/dock/test-plan.pxu

```diff
@@ -2,11 +2,139 @@ id: dock-cert-full
 unit: test plan
 _name: Dock Cert tests
 _description:
- Cert tests for dock devices.
+  Cert tests for dock devices.
 include:
 nested_part:
-  dock-cold-plug-cert
-  dock-hot-plug
+  dock-all-init-cert
+  dock-all-reboot-cert
+  dock-all-hotplug-cert
+  dock-all-suspend-cert
```

Comment:
Maybe this nested part should be at the very end, since the "after suspend" state might affect the beginning of the `dock-all-poweroff-cert` nested part.

Reply:
Yep, you are right. And suspend usually the one cause problem.

Reply:
Done.